### ISSUE TITLE
Support for Marker Data's CSV Format

### ISSUE THREAD

**Issue by IAmVigneswaran** [latenitefilms/MarkerToolbox#33]
It would be a nice addition to support Marker Data's CSV Format. 

[Marker Data Demo_V3.csv](https://github.com/latenitefilms/MarkerToolbox/files/15499211/Marker.Data.Demo_V3.csv)

**Use Case**

We can get somewhat of a fake "Timeline Marker". It is almost like consolidation all the markers from various clips, into a single clip where, users can place it on top of their clips.

1. Users can share a rendered timeline to others along with a `.csv` or `.tsv` extracted from Marker Data.
2. Users then could "import" the markers back using Marker Toolbox.

End result would be something like this -

<img width="1245" alt="Marker Toolbox" src="https://github.com/latenitefilms/MarkerToolbox/assets/118706051/344ce7e9-fa62-41b7-b16b-56d418de8bbe">

<br>

Basically, Marker Toolbox would parse `Marker Name`, `Marker Type`, `Notes` and `Marker Position` columns.

**Comment by orchetect** [comment]
Just for clarity: Marker Data doesn't produce the csv file - MarkersExtractor does, which itself is basically a formatting wrapper around DAWFileKit.

**Comment by IAmVigneswaran** [comment]
> Just for clarity: Marker Data is a GUI frontend for MarkersExtractor. MarkersExtractor outputs the csv/json and is essentially a formatting wrapper around DAWFileKit.

Yes! I believe Chris is aware of the development details. Hence, I skipped the information.
